docs/source/examples/de_sscan.ipynb

```diff
@@ -0,0 +1,1505 @@
+    }
+   ],
+   "source": [
+    "RE(setup_scan1(-1.1, 1.1, 11, 0.2))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Run the scan.  We should not have to reprogram this plan."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "()"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "RE(run_sscan())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Get the scan data.  Same function as before.  It's not a bluesky plan, so `RE()`\n",
+    "is not needed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'m1': array([-1.1 , -0.88, -0.66, -0.44, -0.22,  0.  ,  0.22,  0.44,  0.66,\n",
+       "         0.88,  1.1 ]),\n",
+       " 'I0': array([2., 1., 1., 2., 1., 3., 2., 1., 1., 1., 0.], dtype=float32),\n",
+       " 'diode': array([1., 2., 1., 2., 2., 4., 4., 2., 0., 1., 1.], dtype=float32)}"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "get_sscan_data()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Post data to bluesky as a run\n",
+    "\n",
+    "So far, we have just run `scan1` and shown the data collected.  We'll need to do\n",
+    "a bit more work to get this data into bluesky as a run.  While we might just use\n",
+    "`bps.read(scan1)` to get the data, the data would be named according to the\n",
+    "`scan1` structure.  We want more meaningful names so we re-assign the names of\n",
+    "the original objects (motor and scaler channel names).  To post *bluesky data*,\n",
+    "it must come from an ophyd `Device` (subclass).  We'll create a custom one just\n",
+    "for our setup of `scan1`, then `bps.mv()` each of the arrays.  Since we do not\n",
+    "have timestamps for each of the data points, we'll post the entire array as a\n",
+    "single event.  The event will have the timestamp from the `bps.mv()` plan stub.\n",
+    "\n",
+    "The last five steps are all descriptive.  The run is opened, the `primary`\n",
+    "stream is written with the `scan_data`, then all is buttoned up and bluesky\n",
+    "finished the run.  We define `name=\"scan1\"` so the names of the data in the databroker will match their usage here.  The `scan_data` object is only a tool to get the data from the `sscan` record into the databroker."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ophyd import Component, Device, Signal\n",
+    "\n",
+    "class ThisSscanData(Device):\n",
+    "    m1 = Component(Signal)\n",
```

Comment:
What is `Component`?

Reply:
An ophyd structure. Do not want to repeat this here (it must be explained somewhere else). Should we find and reference the documentation page?

Reply:
`Component` is a tool used by ophyd's `Device` to mark an attribute as an ophyd object that needs to be handled either as a `Device` or a `Signal`.  The first argument tells which type of ophyd (or custom) class to be used.